.github/workflows/node.yml

```diff
@@ -39,14 +42,8 @@ jobs:
 
       - name: Install dependencies & build
         run: |
-          make clean
```

Comment:
If we remove the make clean the js dir will contain the artifacts from the repo, right? So if `npm run build` did nothing the comparison to the repo would still match in the next step.

I'm not sure which of these are changes from the template and which are just cleanup.